lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -36,29 +42,94 @@
             userPreferencesModel = new UserPreferencesModel();
             userPreferencesModel.url = userPreferencesApiUrl;
 
-            if (allowEmailChange) {
+            if (syncLearnerProfileData && enterpriseName) {
+                aboutSectionMessageType = 'info';
+                aboutSectionMessage = HtmlUtils.interpolateHtml(
+                    gettext('Your profile settings are managed by {enterprise_name}. Contact your administrator or {link_start}edX Support{link_end} for help.'),  // eslint-disable-line max-len
+                    {
+                        enterprise_name: enterpriseName,
+                        link_start: HtmlUtils.HTML(
+                            StringUtils.interpolate(
+                                '<a href="{edx_support_url}">', {
+                                    edx_support_url: edxSupportUrl
+                                }
+                            )
+                        ),
+                        link_end: HtmlUtils.HTML('</a>')
+                    }
+                );
+            }
+
+            if (!allowEmailChange || (syncLearnerProfileData && enterpriseReadonlyAccountFields.fields.indexOf('email') !== -1)) {  // eslint-disable-line max-len
                 emailFieldView = {
-                    view: new AccountSettingsFieldViews.EmailFieldView({
+                    view: new AccountSettingsFieldViews.ReadonlyFieldView({
                         model: userAccountModel,
-                        title: gettext('Email Address'),
+                        title: gettext('Email Address (Sign In)'),
                         valueAttribute: 'email',
                         helpMessage: StringUtils.interpolate(
-                            gettext('The email address you use to sign in. Communications from {platform_name} and your courses are sent to this address.'),  // eslint-disable-line max-len
+                            gettext('You receive messages from {platform_name} and course teams at this address.'),  // eslint-disable-line max-len
```

Comment:
Is that message pointing to `current` or `future` state ?

Reply:
This text is provided by UX team.